core/src/main/kotlin/org/evomaster/core/problem/rest/service/AbstractRestSampler.kt

```diff
@@ -81,18 +83,24 @@ abstract class AbstractRestSampler : HttpWsSampler<RestIndividual>() {
         val openApiURL = problem.openApiUrl
         val openApiSchema = problem.openApiSchema
 
+        // set up authentications moved up since we are going to get authentication info from HttpWsSampler
+        setupAuthentication(infoDto)
+
         if(!openApiURL.isNullOrBlank()) {
-            swagger = OpenApiAccess.getOpenAPIFromURL(openApiURL)
+
+            try {
+                retrieveSwagger(openApiURL)
+            }
+            catch (e : Exception) {
```

Comment:
remove the try/catch, and just call `retrieveSwagger(openApiURL)` directly. otherwise you are masking the exception

Reply:
Hi Andrea,

Thanks for your comment. I have fixed this one.